autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/ui/credential/saving/AutofillSavingCredentialsDialogFragment.kt

```diff
@@ -168,7 +200,8 @@ class AutofillSavingCredentialsDialogFragment : BottomSheetDialogFragment(), Cre
 
         onUserRejectedToSaveCredentials()
 
-        pixelNameDialogEvent(Dismissed)?.let { pixel.fire(it) }
+        val onboardingMode = this.view?.findViewById<View>(R.id.keyFeaturesContainer)?.isVisible ?: false
```

Comment:
do you need this `findView` lookup... isn't that what you're referencing with `private lateinit var keyFeaturesContainer: ViewGroup`?